sys/net/crosslayer/ng_pktbuf/ng_pktbuf.c

```diff
@@ -30,6 +30,9 @@
 
 #include "_pktbuf_internal.h"
 
+#define ENABLE_DEBUG (0)
+#include "debug.h"
```

Comment:
I know, but the change to _duplicate also generates a merge conflict if I would add the DEBUG output of it to#3089. So I added it here. Afaik git should be able to handle this kind of conflict here.